mmdet3d/datasets/utils.py

```diff
@@ -37,7 +34,7 @@ def is_loading_function(transform):
             return False
         if obj_cls in loading_functions:
             return True
-        if obj_cls in (MultiScaleFlipAug3D, ):
+        if obj_cls in (MultiScaleFlipAug3D,):
```

Comment:
I do not get this modification, it seems does not change the behavior of the code.